Teams.Tests/ManageTasksServiceTest.cs

```diff
@@ -0,0 +1,95 @@
+﻿using MockQueryable.Moq;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Teams.Data;
+using Teams.Models;
+using Teams.Security;
+using Teams.Services;
+using Task = Teams.Models.Task;
+
+namespace Teams.Tests
+{
+    [TestFixture]
+    class ManageTasksServiceTest
+    {
+        private Mock<IRepository<Task, int>> _tasksRepository;
+        private ManageTasksService _manageTasksService;
+
+        [SetUp]
+        public void Setup()
+        {
+            _tasksRepository = new Mock<IRepository<Task, int>>();
+            _manageTasksService = new ManageTasksService(_tasksRepository.Object);
+            var mock = getFakeDbTasks().AsQueryable().BuildMock();
+            _tasksRepository.Setup(x => x.GetAll()).Returns(mock.Object);
+        }
+
+        [Test]
+        public async System.Threading.Tasks.Task GetAllTasksInTeamAsyncReturnListTasks_ReturnListTasks()
```

Comment:
Forget in names one _ symbol, also bit weird naming
Should be: GetAllTasksInTeamAsync_RepositoryContainsTwoTasksForSelectedTeam_ReturnsTwoTasks

Also name of executed method: GetMyTaskInTeamAsync, better to use GetAllTasksForTeam